xml/System.Text.Json/JsonElement.xml

```diff
@@ -606,7 +606,15 @@ If a property is defined multiple times for the same object, the method matches
       <Docs>
         <summary>Gets a string that represents the original input data backing this value.</summary>
         <returns>The original input data backing this value.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+For a `JsonElement` built from <xref:System.Text.Json.JsonNode>, the value of <xref:System.Text.Json.JsonNode.ToJsonString> is returned.
```

Comment:
Given JsonNode and related types is a new feature (for 5.0) with its design not yet complete, I would withhold making such doc updates until at least the first preview for 5.0 (and where we reflect over the ref assemblies as well).

Reply:
Ok no problem, I'll close this PR.